analyze.py

```diff
@@ -38,16 +38,11 @@ def parse_ticker_dataframe(ticker: list, minimum_date: arrow.Arrow) -> DataFrame
     :param pair: pair as str in format BTC_ETH or BTC-ETH
     :return: DataFrame
     """
-    data = [{
-        'close': t['C'],
-        'volume': t['V'],
-        'open': t['O'],
-        'high': t['H'],
-        'low': t['L'],
-        'date': t['T'],
-    } for t in sorted(ticker, key=lambda k: k['T']) if arrow.get(t['T']) > minimum_date]
-
-    return DataFrame(json_normalize(data))
+    df = DataFrame(ticker) \
+        .drop('BV', 1) \
+        .rename(columns={'C':'close', 'V':'volume', 'O':'open', 'H':'high', 'L':'low', 'T':'date'}) \
+        .sort_values('date')
+    return df[df['date'].map(arrow.get) > minimum_date]
```

Comment:
I'd love to get rid of this filtering based on date, but like you've noticed, the Bittrex query returns way too much data and I can't seem to figure out how the `_` parameter works, currently it seems to make no difference what value you put there. Might be a bug too in the undocumented 2.0 API.

Reply:
Yes their API acts very strange, the data from this endpoint is sometimes slightly outdated for some pairs where there is not much 24h volume (hopefully we can switch to WebSockets in the long term :-) ). I checked this endpoint right now and noticed that their data is not interpolated, do you think we need to interpolate it for TA-lib?

Reply:
I'm not sure what you mean with interpolated here?

Low volume coins are something we should stay away with our bot anyways: no volume means that most TA indicators can't do their job and often no volume means also no volatility which is a must for our bot.

Reply:
With that I mean the intervals from the `tick` do not have a fixed time difference e.g.: 1 minute, but rather a random interval:
```
"T":"2017-09-10T17:42:00"
"T":"2017-09-10T17:50:00"
"T":"2017-09-10T17:54:00"
"T":"2017-09-10T17:59:00"
"T":"2017-09-10T18:05:00"
"T":"2017-09-10T18:09:00"
"T":"2017-09-10T18:19:00"
"T":"2017-09-10T18:29:00"
"T":"2017-09-10T18:35:00"
"T":"2017-09-10T18:39:00"
"T":"2017-09-10T18:44:00"
"T":"2017-09-10T18:49:00"
"T":"2017-09-10T18:54:00"
"T":"2017-09-10T18:59:00"
"T":"2017-09-10T19:04:00"
"T":"2017-09-10T19:09:00"
"T":"2017-09-10T19:14:00"
"T":"2017-09-10T19:21:00"
"T":"2017-09-10T19:29:00"
"T":"2017-09-10T19:34:00"
"T":"2017-09-10T19:44:00"
"T":"2017-09-10T19:49:00"
"T":"2017-09-10T19:55:00"
"T":"2017-09-10T19:59:00"
"T":"2017-09-10T20:04:00"
```
I don't know if TA-lib requires some sort of "fixed interval", I will try it out if it makes a difference if we apply `DataFrame.interpolate`

Yes I agree, it makes no sense to trade low volume coins

Reply:
Ah, that's a good point. We should check that. Good catch!